pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -224,6 +224,70 @@ func guaranteedCPUs(pod *v1.Pod, container *v1.Container) int {
 	if cpuQuantity.Value()*1000 != cpuQuantity.MilliValue() {
 		return 0
 	}
+
+	// The code below works around a bug in the CPUManager, whereby it doesn't
```

Comment:
Nit: This information is _fantastic_. I'm wondering if we should place it in the commit message, and then have a short executive summary (with a reference to more info in the commit message) as a comment in the code? The advantage of the commit message is that it can't go out of date.

Reply:
It's hard to get a reference to the commit message sha in the code because of a chicken/egg problem. We could always refer to the PR or split the code change with the commit message and the comment referring to the commit message into separate commits.

Reply:
Mhhm yes, that is a fair point...

I'll leave this up to you :) I think I slightly prefer splitting the code change with the commit message and then adding the comment referring to the commit message as a separate commit. But its your choice :)